README.md

```diff
@@ -25,11 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="/images/snippet.PNG" alt="logo" width="140"  height="auto" />
-  <br/>
-
   <h3><b>My Portfolio Project - README</b></h3>
 
```

Comment:
I suggest adjusting your Readme file to the project, it is unprofessional to use a readme file from a different project without making the necessary changes to it.